draft-ietf-wimse-s2s-protocol.md

```diff
@@ -586,6 +586,16 @@ TODO: `Workload-Proof-Token` from {{dpop-esque-auth}}
 # Document History
 <cref>RFC Editor: please remove before publication.</cref>
 
+## draft-ietf-wimse-s2s-protocol-01
+
+* Addressed multiple comments from Pieter.
```

Comment:
```suggestion
* Addressed multiple comments from Pieter, co-chair of the WIMSE WG but comments made as an individual.  
```

Reply:
I already merged the PR, will add this text separately.

Reply:
race condition

Reply:
it's ok without too